src/main/java/org/alliancegenome/mati/controller/IdentifierFinder.java

```diff
@@ -43,10 +43,10 @@ public Response find(@HeaderParam("Authorization") final String auth_header, @He
             Long asNumber = Long.parseLong(counter);
             String status;
             if(subdomainSequenceRepository.getValue(subdomainEntity) > asNumber) {
-                status = "active";
+                status = "unassigned";
             }
             else {
-                status = "inactive";
+                status = "assigned";
```

Comment:
I think you may have swapped the conversion: `active` should be `assigned` while `inactive` should be `unassigned`?